data_steward/cdr_cleaner/cleaning_rules/repopulate_person_post_deid.py

```diff
@@ -0,0 +1,129 @@
+"""
+The de-id scripts removes all fields in the person table except for the person_id and the birthdate_time field.
```

Comment:
This module documentation string should have a summary line followed by a blank line before the nice, more verbose description here.  Like:
```
Repopulate the person table from the de-identified observation table.

The de-id scripts remove all fields...
```

Reply:
^^